arbitrum-docs/stylus/stylus-quickstart.md

```diff
@@ -11,209 +11,226 @@ import PublicPreviewBannerPartial from '../partials/_public-preview-banner-parti
+- [rust-analyzer](https://marketplace.visualstudio.com/items?itemName=rust-lang.rust-analyzer): Provides advanced features like smart code completion and on-the-fly error checks
+- [Error Lens](https://marketplace.visualstudio.com/items?itemName=usernamehw.errorlens): Immediately highlights errors and warnings in your code
+- [Even Better TOML](https://marketplace.visualstudio.com/items?itemName=tamasfe.even-better-toml): Improves syntax highlighting and other features for TOML files, often used in Rust projects
+- [Dependi](https://marketplace.visualstudio.com/items?itemName=fill-labs.dependi): Helps manage Rust crate versions directly from the editor
 
-#### Developer wallet / account
+#### Docker
 
-When deploying on and interacting with a testnet chain, it’s important to use a fresh wallet that does not contain any real assets. You’ll often be including private keys as CLI arguments to execute transactions programmatically, **_so avoid using personal accounts for development_**.
+`cargo stylus` requires Docker to operate. You can download Docker from [Docker’s website](https://www.docker.com/products/docker-desktop).
 
-If you’re using [MetaMask](https://metamask.io/), simply click the dropdown at the top middle of the plugin and then click “Add Account” to create a fresh account. It can be helpful to label the account as a dev wallet or “Stylus” for this purpose. You’ll need this newly created account’s private key (as well as some Sepolia ETH) for deploying a smart contract. [Follow the instructions on MetaMask’s website](https://support.metamask.io/hc/en-us/articles/360015289632-How-to-export-an-account-s-private-key) to obtain your key.
+#### Developer wallet/account
 
-![Stylus Wallet](./assets/stylus-wallet.png)
+We recommend creating an account specifically for development that doesn't hold any real assets.
 
-:::caution
+##### Creating a new account
 
-**_Never share your “secret recovery phrase” with anyone_**. Never enter it anywhere. A _private key_ is valid for an individual account only, but a _secret recovery phrase_ can be used to gain access to ALL accounts in your wallet.
+If you’re using [MetaMask](https://metamask.io/), click the dropdown at the top middle of the plugin and then click “Add Account” to create a fresh account.
+Labeling the account as a dev wallet or “Stylus” can be helpful. You’ll need this newly created account’s private key. [Follow the instructions on MetaMask’s website](https://support.metamask.io/hc/en-us/articles/360015289632-How-to-export-an-account-s-private-key) to obtain your key.
 
-:::
+##### Storing your private key
 
-### Testnet ETH
+You might also want to add your developer's private key to your environment variables for easier and safer access.
+To do this, store your hex-encoded private key in a text file and add the path to the file as an ENV variable named `$PRIVKEY_FILE_PATH` to your shell file, e.g.,``.bashrc`.
```

Comment:
```suggestion
To do this, store your hex-encoded private key in a text file and add the path to the file as an ENV variable named `PRIVKEY_FILE_PATH`.
```

I would avoid mentioning adding it to the shell's profile file, since it's not required, and new users might not know how to do this. They can simply do `export PRIVKEY_FILE_PATH=private_key` and that will work for this example.

Reply:
noted, done.